### ISSUE TITLE
Warning about deprecated declaration

### ISSUE THREAD

**Issue by eribertomota** [tgalal/yowsup#1035]
Hi Tarek,

I can see this warning since some releases ago.

WARNING:yowsup.stacks.yowstack:Implicit declaration of parallel layers in a tuple is deprecated, pass a YowParallelLayer instead

Thanks!

**Comment by joaoricardo000** [comment]
Hi, 
When creating a YowStack, just replace the tuple of parallel layers with an YowParallelLayer object, that you create passing the tuple of layers.
Ex.:
```
from yowsup.layers import YowParallelLayer
layers = (RouteLayer,
          YowParallelLayer([YowAuthenticationProtocolLayer,
                            YowGroupsProtocolLayer,
                            YowMessagesProtocolLayer,
                            YowReceiptProtocolLayer,
                            YowAckProtocolLayer,
                            YowMediaProtocolLayer,
                            YowIqProtocolLayer]),
          YowAxolotlLayer,
          ) + YOWSUP_CORE_LAYERS
self.stack = YowStack(layers)
,```

**Comment by tgalal** [comment]
it's coming from the demos, i should have updated them long time ago but I forgot. Will do, thanks

**Comment by dinhoabreu** [comment]
#1087

**Comment by tgalal** [comment]
Fixed by #1036
